sql/2021/structured-data/rdfa_type_ofs.sql

```diff
@@ -0,0 +1,36 @@
+# standardSQL
+  # Count RDFa Type Ofs
+CREATE TEMP FUNCTION
+  getRDFaTypeOfs(rendered STRING)
+  RETURNS ARRAY<STRING>
+  LANGUAGE js AS r"""
+  try {
+    rendered = JSON.parse(rendered);
+    return rendered.rdfa_typeofs.map(typeOf => typeOf.toLowerCase());
```

Comment:
TODO: I should be splitting these at whitespace (e.g. `Book Product` becomes [`Book`, `Product`]).

Reply:
From [Schema.org](https://schema.org/additionalType):

```html
<div vocab="https://schema.org/" resource="#record" typeof="Book Product">
    <h3 property="name">
        Le concerto
    </h3>
</div>
```